reanalysis/comparison.py

```diff
@@ -128,18 +139,108 @@ def common_format_from_results(results_dict: Dict[str, Any]) -> CommonDict:
     return sample_dict
 
 
-def main(results: str):
+def common_format_from_seqr(seqr: str, probands: List[str]) -> CommonDict:
+    """
+    identify the most likely proband for each row, and create a variant object for them
+
+    :param seqr:
+    :param probands:
+    :return:
+    """
+
+    assert seqr.endswith('.tsv'), f'process requires a TSV format export'
+    sample_dict: CommonDict = defaultdict(list)
+
+    with AnyPath(seqr).open() as handle:
+        seqr_parser = DictReader(handle, delimiter='\t')
+
+        sample_cols = [
+            sample_field
+            for sample_field in seqr_parser.fieldnames
+            if SAMPLE_NUM_RE.match(sample_field)
+        ]
+
+        for entry in seqr_parser:
+
+            # get all valid tags
+            tags = [
+                Confidence(tag)
+                for tag in entry['tags'].split('|')
+                if tag in VALID_VALUES
+            ]
+
+            # no relevant tags, not interested...
+            if len(tags) == 0:
+                continue
+
+            # seqr export currently has no notion of 'proband', so find the
```

Comment:
So the logic here is that you will get all affected children? If you are optionally taking multiple family members, why not just take all affected individuals? OR if you are expecting only one individual per family do you need a break after you append to the sample dict?

Reply:
logic changed up, now we gather all affected participants on each row with a non-WT, and all are assigned the described variant, e.g.

- 2 affected children, both het: the commonFormatVariant will be given to both samples
- 2 affected members, one WT & one het: variant only given to het sample